cmd/client/command/common.go

```diff
@@ -189,3 +189,20 @@ func readFromFileOrStdin(specFile string, cmd *cobra.Command) ([]byte, string) {
 
 	return buff, spec.Name
 }
+
```

Comment:
Seems function `readFromFileOrStdin` is useless now, if so, please remove it to keep the code base clear.

Reply:
Good catch.